deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,461 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Installation",
+          "metadata": {
+            "name": "example-installation"
+          },
+          "spec": {
+            "namespacePrefix": "integreatly-",
+            "selfSignedCerts": true,
+            "type": "workshop"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "Installation",
+          "metadata": {
+            "name": "INSTALLATION_NAME"
+          },
+          "spec": {
+            "namespacePrefix": "integreatly-",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "type": "INSTALLATION_TYPE"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integreating greatly
+    certified: "false"
+    containerImage: quay.io/integreatly/integreatly-operator:v1.12.0
```

Comment:
should the version here be v1.13.0?

Reply:
yeah it should. It did update where it mattered, not sure why it didn't update here.

Reply:
pushed a fix now, anyway